pyproject.toml

```diff
@@ -362,6 +362,7 @@ zip = ">=3.0"
 
 [tool.coverage.run]
 source = ["src/pudl/", "test/integration/", "test/unit/"]
+relative_files = true
```

Comment:
Needed to avoid incompatibilities between the coverage reports generated on GitHub and non-GitHub runners, which put the code in different directories in the runner's home directory.